tools/test-proxy/Azure.Sdk.Tools.TestProxy/Azure.Sdk.Tools.TestProxy.csproj

```diff
@@ -21,6 +21,11 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.0" />
   </ItemGroup>
   
+  <ItemGroup Condition="'$(TargetFramework)'=='net7.0'">
```

Comment:
Doesn't look like we need these split out any longer and given we don't support .NET 5.0 any longer we can probably remove that group.